workloads/oai/pkg-example-cucp-bp/config_ran_nf.yaml

```diff
@@ -40,4 +40,4 @@ spec:
       name: oai-nf-config
       namespace: default
     spec:
-      image: "docker.io/oaisoftwarealliance/oai-gnb:2023.w19"
\ No newline at end of file
+      image: "arorasagar/testing-images:oai-gnb-telnet"
```

Comment:
@jain-ashish-sam Can you add a note saying that this image is temporary and later it will be changed? Since this is a personal repository?